plugins/easyimage/styles/easyimage.css

```diff
@@ -19,6 +20,7 @@ For licensing, see LICENSE.md or http://ckeditor.com/license
 
 .easyimage-side {
 	float: right;
+	max-width: 50%;
```

Comment:
Let's make it 25% per your accidental proposal. 25% looks much better. 🙂

Remember to update manual test description.